mesh/profiles/views.py

```diff
@@ -1,3 +1,16 @@
-from django.shortcuts import render
+from django.views.decorators.csrf import csrf_exempt
+from django.http import JsonResponse
 
-# Create your views here.
+@csrf_exempt
+def bio_view(request):
+    if request.method == "POST":
+        data = request.POST
+        if 'accountID' not in data:
+            return JsonResponse({'error': 'Invalid request. Missing accountID field.'}, status=401)
+        if 'biography' not in data:
+            return JsonResponse({'error': 'Invalid request. Missing biography field.'}, status=400)
+        else:
+            print(data.get('accountID') + " " + data.get('biography')) # TODO: Save to database
```

Comment:
Make issue for the TODO link in PR description that this introduces that issue

Reply:
Issue already exists, I've edited the pr description to include it.